### PR TITLE
Add motocross bikes and street-legal dirt bikes

### PR DESCRIPTION
**Pull request by kristleifur**
#### Summary
SUMMARY: Content: "Add motocross and street-legal dirt bikes"

#### Purpose of change
I wanted to add my Husaberg FE 570 to the game. It's a big-bore street-legal dirt bike. Similar to the KTM 500 EXC and 690 Enduro R, or Husqvarna 501 / 701. Around 120kg wet weight, max speed north of 150 kph.

#### Describe the solution
Added a high-powered water-cooled single-cylinder motor. Added some lightweight and fragile motorcycle components to get the weight to an accurate level.

The street-legal bike is 121kg in-game, basically the same as a real plated dirt bike. Top speed is very similar. (The in-game street-legal dirt bike tops out at 144kph. A powerful dirt bike if pushed hard will go up to, say, 160kph.)

Also corrected the offroad motorcycle wheel diameter. The rims are usually 21" in front and 18" in the rear, but the rear tire is thicker so it evens out. Given the wheel sizes in the game 20" seems appropriate.

Also added an MX bike that weighs a bit less and has no headlight or horn.

Finally, there has been a pedal-powered Dirt bike in the game. I only know that term as referring to a combustion-engine powered offroad motorcycle. So renaming the pedal-powered dirt bike to Mountain bike?

#### Additional context
This sort of bike:
BFZQAZNA5QBTNK6RLUY5O6T3S4

In-game:

Screenshot 2019-06-06 at 00 25 23

### Commit messages
* add motocross bike and Street-Legal Dirt Bike (aka enduro bike – the
enduro bike is similar to the mx bike but has a headlight and a stronger
 subframe so it can hold a bit of cargo)
* add mx/enduro motorcycle spawns to some locations
* add modern watercooled big-bore dirt bike engine. 550cc, 55HP, fuel
consumption same as the 50hp V-Twin (a 50hp v-twin would be air-cooled and less
fuel efficient)
* add motorcycle headlight (lightweight and somewhat fragile)
* change dirt bike wheel diameter to 20"
* rename pedal-powered 'Dirt bike' to 'Mountain bike'

### CODE REVIEW

**Comment by mlangsdorf** [comment]
Dirt bike is or used to be American slang for a cheap mountain/off-road bicycle.  If mountain bike is the preferred term, go ahead and change it.

File: data/json/items/vehicle/lights.json [hunk 1]
```diff
@@ -12,6 +12,19 @@
     "volume": 4,
     "price": 3000
   },
+  {
+    "id": "motorcycle_headlight",
+    "type": "GENERIC",
+    "name": "car headlight",
```
Comment (WhiteHusky) [inline data/json/items/vehicle/lights.json]: `"name": "motorcycle headlight"`

Comment (WhiteHusky) [inline data/json/items/vehicle/lights.json]: `"name": "motorcycle headlight"`

**Comment by kevingranade** [comment]
Needs styling.

**Comment by AnthonBerg** [comment]
Thanks all! Am learning as I go! I am committed to needing less assistance and cleanup as I learn more :)

– If the Dirt Bike as pedal powered bike turns out to be a New England idiom then let's switch it back by all means! I don't have a great feeling for geographic idiom use in US English :)

**Pull request merged**